app/src/main/java/home/project/group/financetracker/ui/home/HomeFragment.java

```diff
@@ -55,6 +41,18 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         Fragment fragment = this;
         View root = Theme.themeDecider(inflater, fragment).inflate(R.layout.fragment_home, container, false);
 
+        top5TransTextView = root.findViewById(R.id.top5transTextView);
+
+        top5Transactions = new ArrayList<>();
+        top5Transactions = DatabaseClass.getDatabase(getActivity().getApplicationContext()).getDao().getTop5Transactions();
+
+        top5TransTextView.setText("");
+        /*Inserting the first 5 Names into the ArrayList*/
+        for (int i = 0; i < top5Transactions.size(); i++) {
+            top5TransTextView.append(top5Transactions.get(i).getName());
+        }
+
+
```

Comment:
Don't need this because this is for testing purposes. We only need 'top5Transactions' ArrayList